src/admin-views/editor/panel/fields/dates.php

```diff
@@ -21,6 +21,13 @@
  * @var array<string,mixed> $ticket The ticket.
  */
 
+// Default values for the Start date & time fields.
+$datepicker_format  = Tribe__Date_Utils::datepicker_formats( Tribe__Date_Utils::get_datepicker_format_index() );
+$default_start_date = gmdate( $datepicker_format );
```

Comment:
Do we want to use gmdate here or the site date? (i.e. crossing the date line means this might be set to tomorrow...or yesterday

Reply:
That's a good point. In this scenario `date` should be find since we want to follow site date exactly. I think switching makes sense.